mmengine/registry/registry.py

```diff
@@ -62,11 +62,14 @@ def __init__(self,
                  name: str,
                  build_func: Optional[Callable] = None,
                  parent: Optional['Registry'] = None,
-                 scope: Optional[str] = None):
+                 scope: Optional[str] = None,
+                 locations: List = []):
         from .build_functions import build_from_cfg
         self._name = name
         self._module_dict: Dict[str, Type] = dict()
         self._children: Dict[str, 'Registry'] = dict()
+        self._locations = locations
```

Comment:
Should we make `resgiter_module` support adding new locations? If users add some custom modules in other module, the `_locations` should be updated.

Reply:
The module's code and registry are pre-defined. Currently, there is no such case.